app/src/components/UserProfile.js

```diff
@@ -107,7 +109,7 @@ export default function UserProfile() {
             Zip Code
             <br />
             <input
-              type="text"
+              type="number"
               name="zipCode"
               value={state.zipCode}
               onChange={handleInputChange}
```

Comment:
you can set a `max=99999` attribute (only kind of works), and I wonder if you can have a conditional where `state.zipCode` only stores a value if it is <= 99999 and if the value > 99999 it will just be ignored or the extra number places spliced out?